pkg/controller/replication.go

```diff
@@ -521,6 +531,7 @@ func (s *Service) DeleteLocalVolume(ctx context.Context,
 	}
 	volumeID := splitHandle[0]
 	globalID := splitHandle[1]
+	protocol := splitHandle[2]
```

Comment:
DeleteStorageProtectionGroup() needs to be updated according to protocol. Not sure if that is part of this story's scope.